.env

```diff
@@ -0,0 +1,7 @@
+DOCKER_NGINX_PORT=80:80

```

Comment:
Just add here your external port (left part). Right part can be hardcoded in docker compose

Reply:
@jgrygierek done, please check it now.